app/routes/template/templateAppendix.js

```diff
@@ -32,6 +58,19 @@ router.use('/', async (req, res, next) => {
     });
   }
 
+  // Convert Project ID to ident in GET endpoint
+  if (req.project && req.templateAppendix && req.method === 'GET') {
+    req.templateAppendix.projectId = req.project.ident;
+  }
+
+  // Throw an error if Project ident is provided but not existent
```

Comment:
will there not be an error thrown at line 27 if no project is found for the provided ident?

Reply:
No error will be thrown on line 27 because if no project is found it will return null and we have to manually throw an error. The check happens here because It's where the other error checks are, so I wanted to keep them together for consistency.